lib/gc/ExecutionEngine/CPURuntime/Microkernel/BrgemmOnednn.cpp

```diff
@@ -57,6 +57,14 @@ static std::vector<brgemm_desc_t> g_brgemm_desc_list;
 static std::vector<brgemm_kernel_t *> g_brgemm_kernel_list;
 static std::vector<std::unique_ptr<char[]>> g_brgemm_palette;
 
+struct brgemmInfo {
+  brgemm_desc_t desc;
+  brgemm_kernel_t *kernel;
+  char *palette;
```

Comment:
I suggest not to store raw pointer managed by smart pointer, maybe change to `shared_ptr`?